src/prefect/cli/run.py

```diff
@@ -557,6 +569,12 @@ def run(
         )
     params_dict = {**file_params, **cli_params}
 
+    if schedule and flow_or_group_id:
```

Comment:
Can I assume that checking `if flow_or_group_id` is sufficient to identify that it's intended to be a cloud run?

Reply:
Cloud runs can be done with just `name` as well so it's not sufficient. I would move this to right under the `if path or module` check which is when we are certain we are doing a local run.

Reply:
Gotcha, in this case i've moved the check and updated it to check for name as well:
```python
if schedule and (flow_or_group_id or name):
```